src/index.html

```diff
@@ -16,24 +16,28 @@
     <div class="main" id="app">
         <router-view name="header"></router-view>
         <router-view></router-view>
+        <Popups></Popups>
     </div>
     <script>
         import Vue from 'vue/dist/vue.js';
         import BootstrapVue from 'bootstrap-vue';
+        import Popups from './components/popups';
         import VueRouter from 'vue-router';
         import store from './store/index.js';
         import router from './router/index.js';
         import i18next from 'i18next';
         import Backend from "i18next-node-fs-backend";
         import VueI18Next from '@panter/vue-i18next';
+        import BeetServer from "./lib/BeetServer";
         
 
         
         Vue.use(VueI18Next);
         Vue.use(BootstrapVue);
         Vue.use(VueRouter);
-
+        Vue.component('Popups',Popups);
```

Comment:
Is that any Popup framework or homemade?

Reply:
Not really a framework.

I'm using vue-router for navigating the various screens and wanted a Vue component that remains there regardless of route and doesn't switch. So I have 2 view slots (handled by vue-router: header and the default) and one that stays constant. It's used to display popups & alerts overlaid over whatever route we might be in. I guess you could say it acts as sort of a popup "service"

Reply:
so homemade I guess :P